recipe/meta.yaml

```diff
@@ -27,15 +27,15 @@ requirements:
   run:
     - python >={{ python_min }}
     - click !=8.1.4
-    - cloudpickle >=1.6.0
+    - cloudpickle >=3
     - cryptography >=39.0.1
-    - globus-sdk >=3.3.0
+    - globus-sdk >=3.46.0,<4
     - lazy-object-proxy >=1.6.0
-    - pydantic >=2
+    - pydantic >=2,<3
     # for all python, not just py<3.11, to stay noarch: python
     - tomli
     - tomli-w
-    - typing-extensions >=4.3.0
+    - typing_extensions >=4.3.0
```

Comment:
Let's just go with what grayskull expects so we don't have to keep fighting it.  It's annoying but not worth the maintainer time.